src/main/java/org/wildfly/security/permission/PermissionUtil.java

```diff
@@ -435,13 +435,19 @@ public static Permission createPermission(final Class<? extends Permission> perm
             }
         }
         try {
-            if (twoArg != null) {
+            if (twoArg != null && name != null && actions != null) {
```

Comment:
Well that is problem... in such case I will not obtain info that the problem is in missing argument (or which one?)...